### DISCUSSION TITLE
profile-image-not-displaying

### DISCUSSION THREAD

**Discussion by vanshaj18** [alshedivat/al-folio#539] [Q&A]
Hi! Thanks for an amazing academic webpage. 

I am having trouble displaying the profile image, the profile picture is not displayed and the terminal says the error: **"/vanshaj-kerni/assets/img/me-1400.webp' not found."**

I am not able to understand the error; could someone please help.

system using: ubuntu 20.04 LTS with ruby 2.7.0p0

Thanks

**Comment by vanshaj18** [comment]
So, i searched a bit more....and it seems the fallback for the profile image is not working with the webp image code in the index.html. is there some way out of this?

**Comment by vanshaj18** [comment]
i re downloaded the whole repo again and the images are not displaying in the browser.

**Reply by vanshaj18** [reply]
Today I followed the #501 to install imagemagick from brew. But the problem continues to exists

here is a
>>>> bundler exec jekyll server
Configuration file: /home/vanshajk/vanshaj18.github.io/_config.yml
            Source: /home/vanshajk/vanshaj18.github.io
       Destination: /home/vanshajk/vanshaj18.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
       Jekyll Feed: Generating feed for posts
       Imagemagick: Searching files in /home/vanshajk/vanshaj18.github.io/assets/img/
       Imagemagick: Generating image "/home/vanshajk/vanshaj18.github.io/_site/assets/img/myself-540.webp"
       Imagemagick: Command returned pid 110120 exit 127 with error sh: 1: convert: not found 
       Imagemagick: Generating image "/home/vanshajk/vanshaj18.github.io/_site/assets/img/myself-1400.webp"
       Imagemagick: Command returned pid 110122 exit 127 with error sh: 1: convert: not found 
       Imagemagick: Generated 2 file(s)
         AutoPages: Disabled/Not configured in site.config.

**Reply by KinGeorge** [reply]
Same question here, it seems that when my browser is full screen the image can display, but when not full screen or on my phone, the images fails to display

**Reply by esmjihwankim** [reply]
I'm having the same problem. I'm presuming that there's something wrong with the Imagemagick command 'convert'

**Reply by jrdeng93** [reply]
> 

Do you find a solution so far?

**Comment by rohandebsarkar** [comment]
Check whether ImageMagick is installed and the command `convert` is in your path.

Check the results of `which convert` or `type convert` in your terminal.
